website_sale_preset_carts/models/time_frame.py

```diff
@@ -113,20 +121,88 @@ def adapt_qty_for_household(line):
             self.env['sale.order.line'].create(line)
         return sale_order
 
+    def compute_cart_amount(self, subscriber, lines):
+        amount = [line['price_unit'] * line['product_uom_qty'] * subscriber.nb_household  for line in lines][0]
+        return amount
```

Comment:
You're taking the the first line of the cart as the total cat amount? Seems weird to me. Also, I think `product_uom_qty` is already computed from nb_household.

Reply:
ok fixed I replaced [] by ()